### DISCUSSION TITLE
Error when loading geoTIFF

### DISCUSSION THREAD

**Discussion by el-cordero** [giswqs/geebook#6] [Q&A] [answered]
Hello,

I am getting this error when loading the geoTIFF. What is the issue? Also, I don't believe this changes anything, but there is an added comma after the band values within the vis variable.

`Map = geemap.Map()
URL = 'https://bit.ly/3aSZ0fH'
image = geemap.load_GeoTIFF(URL)
vis = {
    "min": 3000,
    "max": 13500,
    "bands": ["B3", "B2", "B1"],
}
Map.addLayer(image, vis, 'Cloud GeoTIFF')
Map.centerObject(image)
Map`

EEException: Image.loadGeoTIFF: Failed to load the GeoTIFF at 'gs://pdd-stac/disasters/hurricane-harvey/0831/20170831_172754_101c_3B_AnalyticMS.tif#1519335766223629': The overview at index 1 with dimensions (2770, 1307) is not a power-of-two reduction of the image with dimensions (8310, 3919).

**Comment by franegrao** [comment]
I get the same problem. Have you found a solution by chance? 
Not a lot going on in this forum it seems...

**Comment by giswqs** [comment, marked as answer]
It seems the Google Cloud bucket hosting the sample file is no longer accessible. Try the following example instead. 

```python
import geemap
Map = geemap.Map()
URL = 'gs://gcp-public-data-landsat/LC08/01/001/002/LC08_L1GT_001002_20160817_20170322_01_T2/LC08_L1GT_001002_20160817_20170322_01_T2_B5.TIF'
image = geemap.load_GeoTIFF(URL)
vis = {
    "min": 0,
    "max": 20000,
}
Map.addLayer(image, vis, 'Cloud GeoTIFF')
Map.centerObject(image, 6)
Map
```

**Reply by franegrao** [reply]
Thank you.
